src/Widget/Form.php

```diff
@@ -197,6 +211,38 @@ public function enctype(string $value): self
         return $new;
     }
 
+    /**
+     * The <fieldset> HTML element is used to group several controls as well as labels (<label>) within a web form.
+     *
+     * @param bool $value whether the fieldset is enabled or disabled.
+     *
+     * @return static
+     *
+     * @link https://html.spec.whatwg.org/multipage/form-elements.html#the-fieldset-element
+     */
+    public function fieldset(bool $value): self
```

Comment:
I don't think there's a point adding fieldset as a boolean flag that just wraps all the content. See https://jsfiddle.net/6fz3nvLr/ for typical usage. It is multiple fieldsets per form.

Reply:
yes, if it makes sense.

Reply:
fixed.

Reply:
done.